pkg/apis/core/v1alpha1/constants/types_constants.go

```diff
@@ -221,6 +221,8 @@ const (
 	// AnnotationShootKonnectivityTunnel is the key for an annotation of a Shoot cluster whose value indicates
 	// if a konnectivity-tunnel should be deployed into the shoot cluster or not.
 	AnnotationShootKonnectivityTunnel = "alpha.featuregates.shoot.gardener.cloud/konnectivity-tunnel"
+	// AnnotationReversedVPN moves the vpn-server to the seed.
+	AnnotationReversedVPN = "alpha.featuregates.shoot.gardener.cloud/reversed-vpn"
```

Comment:
You could also mention about this annotation in `./docs/usage/reverse-tunnel.md` (or in new usage doc) as right now it is not documented in usage doc.

Reply:
Do you prefer to extend the `reverse-tunnel.md` guide, which currently focuses only on konnectivity tunnel, or would you rather have a separate document?

Reply:
Yep, probably having a separate document sounds better to do not mix both things. However no strong opinion, however you see fit.

Reply:
Added a separate usage document with the latest commit.

Reply:
I think we should try to use `alpha.featuregates.shoot.gardener.cloud/ReversedVPN` - the same name as the feature gate in the configuration. I know that the `KonnectivityTunnel` uses `konnectivitity-tunnel`, but it should make things easier.

Reply:
Not sure, camel case in the annotations is quite odd and also nothing that I've seen in k/k. Anyways, no strong opinion.

Reply:
@mvladev how does this make anything easier? From my perspective, using the current name fits better to the surrounding. Just looking at `types_constants.go`, there are multiple annotations using exactly the same style, i.e. lower case separated with '-'.
No really strong opinion here as well, but I think it would look somewhat weird for this annotation to stand out like this.